### PR TITLE
Feature/GCI-1228: Add skeleton page for check-details

### PR DESCRIPTION
**Pull request by mwejuli**
Adds skeleton page for check-details for certified-copies

### CODE REVIEW

File: src/views/certified-copies/check-details.html [hunk 1]
```diff
@@ -0,0 +1,106 @@
+{% extends "layout.html" %}
+
+{% block pageTitle %}
+  Order a certified document
+{% endblock %}
+
+{% block beforeContent %}
+  {{ govukBackLink({
+    text: "Back",
+    href: "/company/" + companyNumber + "/more"
```
Comment (nabeelamirch) [inline src/views/certified-copies/check-details.html:10]: Back link should be the delivery details

File: src/views/certified-copies/check-details.html [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+  }) }}
+{% endblock %}
+
+{% block content %}
+
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds-from-desktop">
+            <form action="/payment-review" method="post">
+                <h1 class="govuk-heading-xl">Check your order details</h1>
+                <h2 class="govuk-heading-m">Order details</h2>
```
Comment (nabeelamirch) [inline src/views/certified-copies/check-details.html:20]: could you use the nunjucks templates pls

File: src/views/certified-copies/check-details.html [hunk 3]
```diff
@@ -0,0 +1,106 @@
+{% extends "layout.html" %}
+
+{% block pageTitle %}
+  Order a certified document
```
Comment (nabeelamirch) [inline src/views/certified-copies/check-details.html:4]: Does this link go to the correct page?

Comment (mwejuli) [reply src/views/certified-copies/check-details.html:4]: Am not sure I see a link here. You mean the page title?

File: src/views/certified-copies/check-details.html (hunk 2 above)
Comment (mwejuli) [reply src/views/certified-copies/check-details.html:20]: Was hoping to use plain html as it's easier to style and work with..

File: src/views/certified-copies/check-details.html (hunk 1 above)
Comment (mwejuli) [reply src/views/certified-copies/check-details.html:10]: Oh yes..

File: src/views/certified-copies/check-details.html (hunk 2 above)
Comment (nabeelamirch) [reply src/views/certified-copies/check-details.html:20]: The certificates check-details page also uses nunjucks

File: src/views/certified-copies/check-details.html (hunk 3 above)
Comment (nabeelamirch) [reply src/views/certified-copies/check-details.html:4]: Yeee

File: src/views/certified-copies/check-details.html (hunk 2 above)
Comment (mwejuli) [reply src/views/certified-copies/check-details.html:20]: We could improve on that and use plain html for certified-copies -- styling, positioning, working with template variables, etc..  will be that little bit more transparent and easier.

**Pull request merged**